notebooks/image_models/solutions/conditional_gan.ipynb

```diff
@@ -0,0 +1,1190 @@
+{
```

Comment:
<p>Link the previous lab in markdown</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Added. Thanks!</p>